backend/lib/rpc-service/service.js

```diff
@@ -4,14 +4,16 @@ import { encode as cborEncode } from '@ipld/dag-cbor'
 import { rawEventEntriesToEvent } from './utils.js'
 import { Value } from '@sinclair/typebox/value'
 import * as util from 'node:util'
-import { ClaimEvent, RawActorEvent, BlockEvent, RpcRespone } from './data-types.js'
+import { ClaimEvent, RawActorEvent, BlockEvent, RpcResponse, ChainHead } from './data-types.js'
 import pRetry from 'p-retry'
+
 /** @import { Static } from '@sinclair/typebox' */
+/** @import {MakeRpcRequest} from '../typings.d.ts' */
 
 /**
  * @param {string} method
- * @param {Object} params
- * @returns {Promise<object>}
+ * @param {unknown} params
```

Comment:
Let's make it clear that `params` is an array.

```suggestion
 * @param {unknown[]} params
```

Resolved in dffd3cd